operator/src/main/resources/application.properties

```diff
@@ -6,7 +6,7 @@ console.deployment.default-ui-image=quay.io/streamshub/console-ui:${console.depl
 
 quarkus.container-image.build=true
 #quarkus.container-image.group=
-quarkus.container-image.name=console-operator
+quarkus.container-image.name=streamshub-console-operator
```

Comment:
This line changes the name of the image that is pushed. That is, `quay.io/streamshub/console-operator` becomes `quay.io/streamshub/streamshub-console-operator`.

Is it necessary since the scripting changes will modify the name in the YAMLs?

Reply:
Good point, it should remain console-operator as you said.